C/c4Base.cc

```diff
@@ -480,7 +481,7 @@ bool c4_setTempDir(C4String path, C4Error* err) C4API {
     }
 
     
-    sqlite3_temp_directory = (char *)sqlite3_malloc(path.size + 1);
+    sqlite3_temp_directory = (char *)sqlite3_malloc((int)path.size + 1);
```

Comment:
Woo!  Was that a unsigned byte?  Did it overflow?

Reply:
It is an unsigned 64-bit integer.  I have an idea for this on another branch but I consider this sort of thing technical debt.  Here it is probably harmless unless you have got the world's longest path.